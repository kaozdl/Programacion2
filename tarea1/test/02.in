primera línea

3 es el número de esta línea
