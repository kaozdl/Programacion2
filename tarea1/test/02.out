primera línea
Longitud  Texto 
      14  primera línea
mitad < primera línea
La conversión es 3.
El inicio numérico de la palabra leida era 3.
